include/cinder/Utilities.h

```diff
@@ -47,9 +47,9 @@ CI_API fs::path getDocumentsDirectory();
 CI_API void launchWebBrowser( const Url &url );
 	
 //! Returns a vector of substrings split by the separator \a separator. <tt>split( "one two three", ' ' ) -> [ "one", "two", "three" ]</tt> If \a compress is TRUE, it will consider consecutive separators as one.
-CI_API std::vector<std::string> split( const std::string &str, char separator, bool compress = true );
+CI_API std::vector<std::string> split(std::string str, char separator, bool compress = true );
```

Comment:
Why `const std::string&` becomes `std::string`

Reply:
I was testing with some methods that would erase the string but there is no need for it not be const now. Thanks for spotting it.